setup.py

```diff
@@ -29,6 +29,9 @@
         'simplejson',
         'six>=1.4.0',
     ],
+    extras_require={
+        'uwsgi': ['uWSGI>=2.0.14'],
```

Comment:
2.0.14 is the current version that does not support the mule return value yet. I wonder if we could just do uWSGI>2.0.14 and that would pick up the internal post3 version.